ydb/mvp/oidc_proxy/oidc_protected_page.h

```diff
@@ -152,38 +120,10 @@ class THandlerSessionServiceCheck : public NActors::TActorBootstrapped<THandlerS
         return authHeader.StartsWith(IAM_TOKEN_SCHEME);
```

Comment:
Do we need to compare case insensitive here? If client sends `bearer <token>`, is it OK?